db/warehouse/migrate/20241203154146_remove_deprecated_overrides.rb

```diff
@@ -0,0 +1,44 @@
+class RemoveDeprecatedOverrides < ActiveRecord::Migration[7.0]
+  def change
+    safety_assured do
+      drop_project_type_index
+      drop_cols
+    end
+  end
+
+  protected
+
+  def drop_cols
+    [
+      ['Inventory', 'coc_code_override', :string],
+      ['Inventory', 'inventory_start_date_override', :date],
+      ['Inventory', 'inventory_end_date_override', :date],
+      ['Project', 'act_as_project_type', :integer],
+      ['Project', 'hud_continuum_funded', :boolean],
+      ['Project', 'housing_type_override:', :integer],
+      ['Project', 'operating_start_date_override', :date],
+      ['Project', 'operating_end_date_override', :date],
+      ['Project', 'hmis_participating_project_override', :integer],
+      ['Project', 'target_population_override', :integer],
+      ['Project', 'tracking_method_override', :integer], # extra
+      ['ProjectCoC', 'hud_coc_code', :string],
+      ['ProjectCoC', 'geography_type_override', :integer],
+      ['ProjectCoC', 'geocode_override', :string],
+      ['ProjectCoC', 'zip_override', :string],
+    ].each do |table, field, _type|
+      # remove_column table, field, type, if_exists: true
+      rename_column table, field, "deleted_#{field}" if column_exists?(table, field)
```

Comment:
I think the `if column_exists...` should only happen on up, or should check a different name on `down`.

Reply:
good catch, thanks. Should be fixed